spec/requests/articles_spec.rb

```diff
@@ -1,11 +1,78 @@
 require 'rails_helper'
 
 RSpec.describe "Articles", type: :request do
+
+  before(:all) do
+    FactoryBot.create(:article)
+  end
+
   describe "GET article index" do
-    it "returns http success" do
-      get "/articles#index"
+    it "returns success with the articles index template" do
+      get articles_path
+
+      expect(response).to have_rendered(:index)
       expect(response).to have_http_status(:success)
     end
   end
 
+  describe "GET /articles/:id" do
+    it "returns success with the articles index template" do
+      get article_path(1)
```

Comment:
This assumption that the id is 1 for the article you created could lead to brittle tests. I would use `article_path(article)`.